app/models/procedure.rb

```diff
@@ -576,8 +576,13 @@ def clone(admin, from_library)
     procedure.service = nil
 
     transaction do
-      procedure.save
+      if !procedure.valid?
+        procedure.errors.attribute_names.each do |attribute|
+          procedure.send("#{attribute}=", nil)
+        end
+      end
 
+      procedure.save
```

Comment:
On peut en profiter pour faire péter l'erreur ici plutôt que plus tard si jamais il y avait une erreur malgré tout
```suggestion
      procedure.save!
```